templates/index.html

```diff
@@ -15,13 +15,10 @@
 <meta name="twitter:title" content="{{ SITENAME|striptags|e }}">
 {% endif %}
 <meta property="og:url" content="{{ SITEURL }}" />
-{% from '_includes/_defaults.html' import SITE_DESCRIPTION with context %}
-{% if SITE_DESCRIPTION %}
-<meta property="og:description" content="{{SITE_DESCRIPTION|e}}" />
-<meta name="twitter:description" content="{{SITE_DESCRIPTION|e}}">
-{% endif %}
 <meta property="og:site_name" content="{{ SITENAME|striptags|e }}" />
 <meta property="og:article:author" content="{{ AUTHOR }}" />
+<!-- Fill in Site Keywords-->
```

Comment:
Do we have keywords for a site? To my understanding, keywords are for an article or page but not a whole site. Can you please confirm?

Reply:
My understanding is that it's for the keywords we can use to search for the site, so the 'main page' should be containing all the keywords from all the articles.